Sources/TimeStorage.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+
+/// Defines where the user defaults are stored
+public enum TimeStoragePolicy {
+    /// Uses `UserDefaults.Standard`
+    case standard
+    /// Attempts to use the specified App Group ID (which is the String) to access shared storage.
+    case appGroup(String)
+
+    public init(appGroupID: String?) {
+        if let appGroupID = appGroupID {
+            self = .appGroup(appGroupID)
+        } else {
+            self = .standard
+        }
+    }
+}
+
+public class TimeStorage {
+    private var userDefaults: UserDefaults
+    private let kDefaultsKey = "KronosStableTime"
+
+    private var _stableTime: TimeFreeze?
```

Comment:
Sorry I don't mean to have a cache on a different variable,  I'd go back to what you have (I didn't realize we actually didn't change the behavior since the state is in the other class)

sorry again!

Reply:
Reverted. No worries!